pandas/tests/tools/test_to_timedelta.py

```diff
@@ -210,3 +210,18 @@ def test_to_timedelta_nullable_int64_dtype(self):
         result = to_timedelta(Series([1, None], dtype="Int64"), unit="days")
 
         tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        ("input", "expected"),
+        [
+            ("8:53:08.71800000001", "8:53:08.718"),
+            ("8:53:08.718001", "8:53:08.718001"),
+            ("8:53:08.7180000001", "8:53:08.7180000001"),
```

Comment:
can you add a case with a negative timedelta

Reply:
Done